report.lisp

```diff
@@ -92,7 +92,9 @@
                   (or (ignore-errors (format-result result :oneline))
                       result)
                   err)
-            (setf (status result) :failed)))
+            (setf (status result) :failed)
+            (unless *silence-compilation-errors-p*
+              (error err))))
```

Comment:
This re-signals errors, which destroys stack trace information. This needs to be reformulated using `handler-bind` instead to avoid re-signalling if the flag is set.

Reply:
Not sure how to use `handler-bind` to accomplish this, fixed the issue in a different manner for now.